valet

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 SOURCE="${BASH_SOURCE[0]}"
+SUDOCMDS="install start stop secure share"
```

Comment:
Are the changes in this file related to this PR or were they added by mistake? If intentional, can you describe the changes in more detail? Thanks 😊